lib/Mojo/Util.pm

```diff
@@ -379,6 +381,25 @@ sub unquote {
   return $str;
 }
 
+sub urandom_bytes {
+  my $num = shift || 32;
+
+  return Crypt::URandom::urandom($num) if CRYPT_URANDOM;
+
+  croak 'Cannot find /dev/urandom, install Crypt::URandom from CPAN' unless -e '/dev/urandom';
```

Comment:
Is it worth having a fallback to read `/dev/urandom` vs simply requiring the user to have this module installed if they are using this feature?